ultraplot/constructor.py

```diff
@@ -762,148 +773,182 @@ def _pop_modification(key):
     return cmap
 
 
-def Cycle(*args, N=None, samples=None, name=None, **kwargs):
-    """
-    Generate and merge `~cycler.Cycler` instances in a variety of ways.
+class Cycle(cycler.Cycler):
+    def __init__(self, *args, N=None, samples=None, name=None, **kwargs):
```

Comment:
Doc strings for classes are always added to the class, not the __init__ function.